src/Parallel/Info.hpp

```diff
@@ -0,0 +1,101 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines functions that provide low-level system information such as number
+/// of nodes and processors, in a way that they are mockable in ActionTesting.
+
+#pragma once
+
+// The functions in namespace `sys` that return information on nodes
+// and cores are templated on DistribObject.  Actions should use
+// these functions rather than the raw charm++ versions (in the
+// sys::charm namespace below) so that the mocking framework will see
+// the mocked cores and nodes.
```

Comment:
Consider making this Doxygen.

Reply:
also there's a residual `sys::charm` -> `sys` in line 13